Sources/App/Extensions/String+ExpandingTilde.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  File.swift
+//  
+//
+//  Created by Julian Gentges on 10.08.20.
+//
+
+import Foundation
+
+extension String {
+    public var expandingTildeInPath: String {
+        var path = NSString(string: self).expandingTildeInPath
+        
+        if self.hasSuffix("/") {
```

Comment:
Since we add the trailing slash within this function, the expandingTildeInPath method will return a correct value.

Reply:
This does not really add a slash. It only ensures that the output path has a trailing slash if the input path had a slash.
But if the original path had no slash, no slash gets added.

Reply:
So as I mentioned this in the previous comment my idea is something like this:

Environment.path("BASE_PATH") -> this should return the proper path including the trailing slash & tilde resolution

```swift
extension Environment {
   static func path(_ key: String) -> String {
//      use .fetch
//      resolve ~ if necessary
//      append '/' if necessary
   }
}

If you agree with this approach (IMHO it's way better & safer than our original idea) could you please alter the PR one more time according to this so I can merge to master. Thanks again for your help. 👍